### ISSUE TITLE
Problem reproducing tutorial

### ISSUE THREAD

**Issue by iimog** [jakelever/kindred#15]
Following the tutorial I had the following problem:

```
$ python annotate.py --corpus corpus.txt --wordlists city.txt,country.txt --outDir annotations
Setting up output directory
Loading and parsing corpus:
Splitting the corpus into sentences so that we can save any annotated sentences and don't need to annotate it all
Loading wordlists:
  city - city.txt
  country - country.txt
Annotating entities in corpus with wordlists
Finding all candidate relations
Time to through some of the candidate relations and annotate some...

For each sentence, choose an existing option or type the name of a new annotation

Saving annotated corpus of 18 sentences (with relations that you have just annotated)
Saving unannotated corpus of 0 sentences (which you did not review)
```

So it did not ask for a single candidate relation. It seems like no entity was annotated at all. Digging a little bit in the source code it appears that tokens are lowercased before matching while the keys in the lookup dictionary might remain capitalized. So I tried it with lowercased wordlists:

```
$ cat city.txt | tr A-Z a-z >city_lc.txt
$ cat country.txt | tr A-Z a-z >country_lc.txt
$ python annotate.py --corpus corpus.txt --wordlists city_lc.txt,country_lc.txt --outDir annotations_lc
Setting up output directory
Loading and parsing corpus:
Splitting the corpus into sentences so that we can save any annotated sentences and don't need to annotate it all
Loading wordlists:
  city_lc - city_lc.txt
  country_lc - country_lc.txt
Annotating entities in corpus with wordlists
Finding all candidate relations
Time to through some of the candidate relations and annotate some...

For each sentence, choose an existing option or type the name of a new annotation

############################## (1/22)
Paris is the capital of France.
x:Done 0:None ? is_capital

############################## (2/22)
Birmingham is a city in the United Kingdom.
x:Done 0:None 1:is_capital ? 0

############################## (3/22)
The capital city of Spain is Madrid.
x:Done 0:None 1:is_capital ? 1

############################## (4/22)
Los Angeles, a large city in the United States, is home to Hollywood.
x:Done 0:None 1:is_capital ? 
```

So this appears to fix the issue.

My versions are:
```
$ uname -a
Linux ee02bde556c7 4.19.44 #1-NixOS SMP Thu May 16 17:41:32 UTC 2019 x86_64 GNU/Linux
$ python --version
Python 3.7.3
$ pip show kindred
Name: kindred
Version: 2.5.2
Summary: A relation extraction toolkit for biomedical text mining
Home-page: http://github.com/jakelever/kindred
Author: Jake Lever
Author-email: jake.lever@gmail.com
License: MIT
Location: /usr/local/lib/python3.7/site-packages
Requires: six, numpy, docstringtest, networkx, scipy, intervaltree, fasteners, scikit-learn, future, sphinx, bioc, spacy, sphinx-rtd-theme, lxml, pytest-socket
Required-by: 
$ pip show spacy
Name: spacy
Version: 2.1.4
Summary: Industrial-strength Natural Language Processing (NLP) with Python and Cython
Home-page: https://spacy.io
Author: Explosion AI
Author-email: contact@explosion.ai
License: MIT
Location: /usr/local/lib/python3.7/site-packages
Requires: wasabi, jsonschema, numpy, murmurhash, requests, srsly, blis, cymem, preshed, thinc, plac
Required-by: kindred, en-core-web-sm
```

**Comment by iimog** [comment]
The same problem exists with `Python 2.7.16`

**Comment by jakelever** [comment]
Hey, thanks for identifying this! The case insensitivity was lost in a recent change to EntityRecognizer and the tests didn't cover it. I've now fixed it and updated to v2.5.3 plus added a new test for the tutorial code. Hopefully your fix isn't necessary any more.

**Comment by iimog** [comment]
Thanks a lot for the quick fix. I can confirm that the issue is fixed in v2.5.3. Also thanks for the great software.
